src/lightning_trainable/trainable.py

```diff
@@ -102,7 +105,11 @@ def configure_optimizers(self):
                     interval="step"
                 )
             case None:
-                lr_scheduler = None
+                lr_scheduler = dict(
```

Comment:
I think it is cleaner to return `[optimizer], lr_schedulers` below and set `lr_schedulers = []` if none is specified

Reply:
We can also just return `optimizer` when no `lr_scheduler` is specified, and return a `dict` containing both otherwise.